components/core/Server.py

```diff
@@ -0,0 +1,46 @@
+from flask import Flask
+from flask.ext.cors import CORS
+from flask import send_file, send_from_directory
+from flask import request, jsonify, abort, Response, g
+from flask_socketio import SocketIO, join_room
+from Auth import *
+from Models import *
+from DownloadManager import *
+import json, urllib.request, urllib.error, urllib.parse, os, _thread
+from multiprocessing import Process
+from DownloadDaemon import starter
+from EMail import send_mail
+import sys
+from gevent import monkey
+
+monkey.patch_all(ssl=False)
+
+server = Flask(__name__)
+server.config['SECRET_KEY'] = "123456789"
```

Comment:
hardcoded secret?

Reply:
I think it's kind of private key for verifying auth tokens. It can be kept on some secret files but it would be hardcoded anyway, right? @agentmilindu @rehrumesh

Reply:
Yes, this secret should not be hardcoded like this in the code. It should go to the `.env` file.
But because this PR is about syncing the commits, shall we merge this and add a separate bug to fix this?

Reply:
Yes. Lets procced like that.